frontend/src/components/common/form/ApplicationSelect.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import { ref, type PropType, onMounted } from 'vue';
+import Dropdown, { type DropdownChangeEvent } from 'primevue/dropdown';
+import type { FamApplication } from 'fam-app-acsctl-api';
+
+const props = defineProps({
+    label: { type: String, required: false, default: 'Select application:' },
+    model: {
+        type: (Object as PropType<FamApplication>) || null,
+        required: true,
+    },
+    options: {
+        type: Array<FamApplication>,
+        default: [],
+        required: true,
+    },
+    placeholder: {
+        type: String,
+        required: false,
+        default: 'Choose an application',
+    },
+});
+
+const model = ref<FamApplication>();
```

Comment:
This is a nice suggestion but the defineModel() was first introduced in `Vue 3.3`. I believe we are currently using `Vue 3.2.38`, maybe it could be refactored in the future when we bump our Vue version.

Reply:
If the new ApplicationSelect component is working fine, and if you prefer, then can do that in additional ticket.
Our frontend uses `3.4.6` currently, so it is available.

Reply:
Oh, I see. Could I update the Vue version number in `package.json`? I got confused by it. @MCatherine1994

Reply:
Thanks for the clarification everyone!